folium/folium.py

```diff
@@ -49,6 +52,8 @@
      'https://cdnjs.cloudflare.com/ajax/libs/Leaflet.awesome-markers/2.0.2/leaflet.awesome-markers.css'),  # noqa
     ('awesome_rotate_css',
      'https://rawgit.com/python-visualization/folium/master/folium/templates/leaflet.awesome.rotate.css'),  # noqa
+    ('beautify_marker_css',
```

Comment:
Take a look at the other plugins to see how to inject this `css` only when the `BeautifyMarker` is used and not globally.